aten/src/ATen/native/native_functions.yaml

```diff
@@ -7794,7 +7794,7 @@
   dispatch:
     CompositeExplicitAutograd: masked_scatter
 
-- func: masked_scatter_backward(Tensor grad, Tensor mask, SymInt[] sizes) -> Tensor
+- func: masked_scatter_backward(Tensor grad_out, Tensor mask, SymInt[] sizes) -> Tensor
```

Comment:
This ia a BC-breaking change and I don't think it's worth going through the whole deprecation process for this rename. Can you please keep the original name here?

Reply:
Ho realized that I was only seeing the diff from the latest commit. Ignore this :D